rpc/common/admin/uptime/moc_common_uptime.proto

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache v2.0 license.
+
```

Comment:
Quick feedback is, I see folks are implementing grpc service for each request.

This is more like AgentInfo and can have more properties. We could later merge version to this.
Can you please rename UptimeRequest to AgentInfo request

Reply:
Note that the HealthAgent service has a method for getting AgentInfo: https://github.com/microsoft/moc/blob/main/rpc/common/admin/health/moc_common_health.proto#L32
Would we want to include this property in the AgentInfoResponse on this service? I think it would make sense, but as I noted in a previous comment, the cloud agent doesn't currently implement this API and at some point we'd need to add support for the uptime property to the other services that implement this API.